content/en/blog/falco-0-36-1/index.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Falco 0.36.1
+date: 2023-10-16
+author: Andrea Terzolo, Luca Guerra
+slug: falco-0-36-1
+tags: ["Falco","Release"]
+---
+
+Today we announce the release of **Falco 0.36.1** 🦅!
+
+## Fixes
+
+Here is a tiny patch release! It addresses some small bugs that will not bother us and our users anymore:
```

Comment:
"Falco's 0.36.1 release is a small patch aimed at protecting our uses by addressing a few minor bugs"